frontend/src/views/WasteActionsPage/WasteActionPage.vue

```diff
@@ -0,0 +1,162 @@
+<template>
+  <div id="wasteaction-page">
+    <div v-if="wasteaction">
+      <BreadcrumbsNav :title="wasteaction.title" :links="[{ to: { name: 'WasteActionsHome' } }]" />
+      <v-row>
+        <img
+          :src="
+            wasteaction.lead_image
+              ? wasteaction.lead_image.meta.download_url
+              : '/static/images/wasteaction-default-image.jpg'
+          "
+          class="lead-image"
+          alt=""
+        />
+      </v-row>
+      <v-row>
+        <v-col cols="12" class="mt-8" sm="2">
+          <v-row justify="end">Type d'action</v-row>
+          <v-row justify="end">
+            <DsfrTag :text="effortLabel" :closeable="false" color="rgb(238, 238, 238)" :small="true" />
+          </v-row>
+          <v-row justify="end">
+            <p class="text-right">
+              <i>{{ effortDescription }}</i>
+            </p>
+          </v-row>
+          <v-row justify="end" class="mt-12">Origine du gaspillage</v-row>
+          <v-row justify="end">
+            <DsfrTagGroup
+              v-if="wasteaction.waste_origin.length"
```

Comment:
```suggestion
              v-if="wasteOrigins.length"
```
dans l'esprit du ["Principle of least astonishment"](https://softwareengineering.stackexchange.com/questions/187457/what-is-the-principle-of-least-astonishment) on essaye d'être coherent sauf si il y a une raison autrement. Dans ce cas, tu donnes `wasteOrigins` à `tags` en dessous, alors pourquoi pas le reutiliser ici